Fable.sln

```diff
@@ -34,6 +34,8 @@ Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "fable-compiler-js", "src\fa
 EndProject
 Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Fable.AST", "src\Fable.AST\Fable.AST.fsproj", "{7D98854F-8786-4F4B-9B57-2C3F905BB53D}"
 EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "fcs-fable", "src\fcs-fable\fcs-fable.fsproj", "{BA15613F-1C85-47F5-8A55-794D3BC98F3C}"
```

Comment:
I'm not sure if this is the right thing to do, but I couldn't build in Visual Studio 2019 because it was complaining about this project reference missing from the solution...

Reply:
I would prefer not to include it in the solution. It's a big project that takes time to load, and the only reason it's in this repo is to make the CI build easier when self-compiling FCS+Fable. If VS complains too much maybe we can remove Fable.Standalone from the solution, I think it's the only one which references this.

Reply:
Alright, I will remove this commit from the PR. It's not too hard to keep it only in my local repo if I need it.